src/components/RequiredMark/RequiredMark.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { IonText } from "@ionic/react";
+
+const RequiredMark: React.FC = () => {
```

Comment:
![shocked-magikarp-omg](https://github.com/microbiomedata/nmdc-field-notes/assets/134325062/a129d8ad-859b-4789-b804-bacb1710eeff)

Reply:
I will nominate you for a Nobel Prize in aster`*`...ophysics!